iFixit/react-commerce
# Refactor page wrapper

closes #1360 

This PR is introducing a new Wrapper component that substitutes the legacy PageContentWrapper component used in product and collection pages.
The new wrapper is needed because we are working on the new product list page described in https://github.com/iFixit/design/issues/510

There are still a couple of sections to adapt like footer, communityNext page etc. Given that they defined their own wrapper and knowing that there is some work ongoing there we decided to postpone those parts and focus only on product and collection page main content

### QA

1. Visit [Vercel preview](https://react-commerce-git-refactor-wrapper-ifixit.vercel.app/Parts)
2. Verify that all sections adhere to the new wrapper sizes. To help checking this out, all optional Strapi sections are already enabled on the `/Parts` collection. To check product page a product page like [this](https://react-commerce-git-refactor-wrapper-ifixit.vercel.app/products/iphone-11-pro-screen), very rich in sections should help
